extensions/screen-studio/src/record-area.tsx

```diff
@@ -0,0 +1,6 @@
+import { ActionPanel, Action, Icon, List, open } from "@raycast/api";
+
+export default async function Command() {
+  await open("screen-studio://record-area");
```

Comment:
Feature request for the future 😇 One thing I'd personally really like is to be able to save specific areas, so I could easily start an area recording from a saved preset

```
open("screen-studio://record-area?w=500&h=800&x=0&y=0")
```

Then this extension could let users create them and easily start a recording via a Hotkey.

Could also be implemented directly in Screen Studio and exposed in Raycast

Reply:
I agree, but I think I'll add it at later time. I'll need to first figure out some good UX around it in the app itself.